library/core/src/ascii/ascii_char.rs

```diff
@@ -557,6 +514,22 @@ impl [AsciiChar] {
     }
 }
 
+#[unstable(feature = "ascii_char", issue = "110998")]
+impl From<AsciiChar> for u8 {
+    #[inline]
+    fn from(chr: AsciiChar) -> Self {
+        chr as u8
+    }
+}
+
+#[unstable(feature = "ascii_char", issue = "110998")]
+impl From<AsciiChar> for char {
+    #[inline]
+    fn from(chr: AsciiChar) -> Self {
+        chr as u8 as char
+    }
+}
+
```

Comment:
These `From` impls seem reasonable, but please document these in the PR description/title.